--- setup.py    2021-01-26 17:15:46.490132912 +0100
+++ setup-patch.py      2021-01-26 17:16:33.030133153 +0100
@@ -71,9 +71,12 @@ setuptools.setup(
             "topics.json",
         ],
     },
+    entry_points = {
+        'console_scripts': ['mautrix-facebook=mautrix_facebook.__main__:main']
+    },
     data_files=[
-        (".", ["alembic.ini", "mautrix_facebook/example-config.yaml"]),
-        ("alembic", ["alembic/env.py"]),
-        ("alembic/versions", glob.glob("alembic/versions/*.py"))
+        ("./share/mautrix-facebook/", ["alembic.ini", "mautrix_facebook/example-config.yaml"]),
+        ("./share/mautrix-facebook/alembic", ["alembic/env.py"]),
+        ("./share/mautrix-facebook/alembic/versions", glob.glob("alembic/versions/*.py"))
     ],
 )