--- setup.py    2021-01-26 17:14:05.400132400 +0100
+++ setup-patch.py      2021-01-26 17:13:32.970132230 +0100
@@ -69,9 +69,12 @@ setuptools.setup(
         "web/public/*.mako", "web/public/*.png", "web/public/*.css",
         "example-config.yaml",
     ]},
+    entry_points = {
+        'console_scripts': ['mautrix-telegram=mautrix_telegram.__main__:main']
+    },
     data_files=[
-        (".", ["alembic.ini", "mautrix_telegram/example-config.yaml"]),
-        ("alembic", ["alembic/env.py"]),
-        ("alembic/versions", glob.glob("alembic/versions/*.py"))
+        ("./share/mautrix-telegram", ["alembic.ini", "mautrix_telegram/example-config.yaml"]),
+        ("./share/mautrix-telegram/alembic", ["alembic/env.py"]),
+        ("./share/mautrix-telegram/alembic/versions", glob.glob("alembic/versions/*.py"))
     ],
 )